### PR TITLE
RBAC clarity for application lifecycle

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/8940

### CODE REVIEW

**Comment by dockerymick** [comment]
@drkho @ckandag When yall have an opportunity, may you both review this PR

**Comment by drkho** [comment]
@dockerymick Can we also update this section please for the requirements of channel namespace? https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.1/html/security/security#application-lifecycle-RBAC

The channel namespace's role is not clear in this section.
Screen Shot 2021-02-21 at 11 31 46 AM

**Comment by ckandag** [comment]
@dockerymick  : Roke @rokej  is probably better suited to review RBAC around subscriptions than me

File: security/rbac.adoc [hunk 1]
```diff
@@ -162,9 +162,9 @@ View the following console and API RBAC tables for cluster lifecycle:
...
+When you create an application, the `subscription` namespace is created and the configuration map is created in the `subscription` namespace. You must also have access to the `channel` namespace. When you want to apply a subscription, you must be a subscription administrator. For more information on managing applications, see link:../manage_applications/managing_subscriptions.adoc#creating-and-managing-subscriptions[Creating and managing subscriptions].
 
-To perform application lifecycle tasks, users must have access to the namespace where the application is created and to the `managedcluster` namespace. For example, the required access to create applications in namespace "N" is a namespace-scoped binding to the `admin` role for namespace "N".
+To perform application lifecycle tasks, users must have access to the `open-cluster-management/application` namespace and to the `managedcluster` namespace. For example, the required access to create applications in namespace "N" is a namespace-scoped binding to the `admin` role for namespace "N".
```
Comment (rokej) [inline security/rbac.adoc]: `open-cluster-management/application` is not a name. (namespace where the application is created) so should it be in single quotes? 

Similarly `managedcluster` is not the name of the namespace. The actual namespace name is the name of the managed cluster.

Comment (dockerymick) [reply security/rbac.adoc]: @rokej Single tick marks are probably not needed. Instead I brought emphasis to the namespaces by using italics. For example, I changed `managedcluster` to _managed cluster_

**Comment by dockerymick** [comment]
@rokej I updated the PR based on the suggestions from our Slack conversation. When you have an opportunity, may you review this PR

**Review by rokej — APPROVED** [review]

**Pull request merged**